### PR TITLE
Add method that deletes product and create the general product set

### PR DESCRIPTION
**Pull request by OlagunjuPhillips**

### CODE REVIEW

File: project/src/main/java/com/google/sps/servlets/BrowseServlet.java [hunk 1]
```diff
@@ -114,8 +114,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     searchInfo.setProperty("imageUrl", null);
     searchInfo.setProperty("textSearch", null);
     if (userUploadedImage) {
-      Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
-      String gcsUrl = CloudStorageLibrary.getGcsFilePath(files);
+    //   Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
```
Comment (mrjwashiv) [inline project/src/main/java/com/google/sps/servlets/BrowseServlet.java]: you can probably delete this

File: project/src/main/java/com/google/sps/servlets/ViewProductsServlet.java [hunk 2]
```diff
@@ -115,8 +115,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     searchInfo.setProperty("imageUrl", null);
     searchInfo.setProperty("textSearch", null);
     if (userUploadedImage) {
-      Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
-      String gcsUrl = CloudStorageLibrary.getGcsFilePath(files);
+    //   Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
```
Comment (mrjwashiv) [inline project/src/main/java/com/google/sps/servlets/ViewProductsServlet.java]: Same here

File: project/src/main/java/com/google/sps/servlets/CloudVisionServlet.java [hunk 3]
```diff
@@ -114,12 +114,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the URL of the image that the user uploaded.
 
     Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
-    String gcsUrl = CloudStorageLibrary.getGcsFilePath(files);
+    String gcsUrl = CloudStorageLibrary.getGcsFilePath(request, blobstore);
     // String gcsUrl = CloudStorageLibrary.getGcsFilePath(request, blobstore);
```
Comment (mrjwashiv) [inline project/src/main/java/com/google/sps/servlets/CloudVisionServlet.java]: Same here

File: project/src/main/java/com/google/sps/servlets/CloudVisionServlet.java [hunk 4]
```diff
@@ -114,12 +114,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the URL of the image that the user uploaded.
 
     Map<String, List<FileInfo>> files = blobstore.getFileInfos(request);
```
Comment (mrjwashiv) [inline project/src/main/java/com/google/sps/servlets/CloudVisionServlet.java]: You can delete this

**Review by mrjwashiv — CHANGES_REQUESTED** [review]

**Review by mrjwashiv — APPROVED** [review]

**Review by jayspringfield — APPROVED** [review]
Looks good!

**Pull request merged**